myproject/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for myproject project.
+
+Generated by 'django-admin startproject' using Django 5.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-)8*s^$dx-oz02k8%*=dzt09**_7jt*i!tvmzr7z8_d84&!jw^g'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'myapp.apps.MyappConfig',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
```

Comment:
```suggestion
    'django.contrib.staticfiles',
    'rest_framework',
```

@sameerbagga looks like you might have missed committing something regarding `settings.py`, or maybe stripped something out during rebase?

QA Before:
<img src="https://github.com/sameerbagga/descartes/assets/59978617/b8f919ba-e210-468d-bb19-515c19b83a9b" width="200">
<img src="https://github.com/sameerbagga/descartes/assets/59978617/28d09656-6c84-40db-8935-95d75c0f7225" width="200">


QA After:
<img width="200" alt="Screenshot 2023-12-21 at 4 59 56 PM" src="https://github.com/sameerbagga/descartes/assets/59978617/03d98c2c-aeed-4d21-81f4-eb1d82c8b263">

Reply:
@aeul-groundcloud Looks like DRF (Django Rest Framework) was setup by default to render HTML. I made the change in settings.py (line 106) to change that to JSON rendering.
Sorry I forgot to test the app on browser, but only used POSTMAN initially. It should be working fine now on both browser and POSTMAN. Please confirm.

Reply:
All good! Works great now, thanks!

Reply:
Perfect. Thanks for giving this a look even so late :)

Reply:
Django applications/plugins should always be added to INSTALLED_APPLICATIONS.

There are other things that it does besides adding to the template load path, such as calling the ready hook.
(code that should run after django has finished bootstrapping, as opposed to module import time)
Not every application does something that would cause it to hard error if it wasn't added, but they.

Some django application/plugin ready examples
https://github.com/encode/django-rest-framework/blob/0f39e0124d358b0098261f070175fa8e0359b739/rest_framework/apps.py#L10
https://github.com/django-webpack/django-webpack-loader/blob/6d77462e28f2dea77b3ec143dd67b3198cea0231/webpack_loader/apps.py#L27-L29
https://github.com/openwisp/django-rest-framework-gis/blob/5840e4fa2747414c4407fe11acf31260c1d6ab43/rest_framework_gis/apps.py#L7-L35

Reply:
Great catch boss. Thanks for letting me know about this one. I'm not sure how I missed it.
Got this one fixed now with the new commit. I realized that after including rest_framework under INSTALLED_APPS, I didn't even need to disable HTML rendering anymore.

Reply:
no problem